install/install_cassandra.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# The contents of this file are subject to the Common Public Attribution
+# License Version 1.0. (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://code.reddit.com/LICENSE. The License is based on the Mozilla Public
+# License Version 1.1, but Sections 14 and 15 have been added to cover use of
+# software over a computer network and provide for limited attribution for the
+# Original Developer. In addition, Exhibit A has been modified to be consistent
+# with Exhibit B.
+#
+# Software distributed under the License is distributed on an "AS IS" basis,
+# WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+# the specific language governing rights and limitations under the License.
+#
+# The Original Code is reddit.
+#
+# The Original Developer is the Initial Developer.  The Initial Developer of
+# the Original Code is reddit Inc.
+#
+# All portions of the code written by reddit are Copyright (c) 2006-2015 reddit
+# Inc. All Rights Reserved.
+###############################################################################
+set -x -e
+
+APTITUDE_OPTIONS=${APTITUDE_OPTIONS:-"-y"}
+export DEBIAN_FRONTEND=noninteractive
+
+# add the datastax cassandra repos
+echo deb http://debian.datastax.com/community stable main | sudo tee /etc/apt/sources.list.d/cassandra.sources.list
+wget -qO- -L https://debian.datastax.com/debian/repo_key | sudo apt-key add -
+
+# grab the new ppas' package listings
+sudo apt-get update
```

Comment:
it's kinda a bummer that we'll be calling `apt-get update` so many times now. that's quite slow.